src/Model/Actions.re

```diff
@@ -61,6 +59,7 @@ type t =
   // Reload keybindings from configuration
   | KeyBindingsReload
   | KeyBindingsParseError(string)
+  | KeybindingInvoked({command: string})
```

Comment:
This was added to distinguish commands invoked by keybindings from the stringly-typed `Command` action that actually runs the command, which should be replaced by a strongly typed variants and scoped to where the command is defined.